hieradata/common.yaml

```diff
@@ -606,6 +606,10 @@ hosts::production::frontend::hosts:
     ip: '10.1.2.2'
   frontend-2:
     ip: '10.1.2.3'
+  draft-email-campaign-frontend-1:
```

Comment:
You'll need to define IP addresses for each environment if you wish to add these machines to each environment.